ihmc-perception/src/main/java/us/ihmc/perception/detections/yolo/YOLOv8DetectionResults.java

```diff
@@ -17,17 +15,13 @@
 
 public class YOLOv8DetectionResults
 {
-   private final static Stopwatch totalWatch = new Stopwatch();
-   private final static Stopwatch floatWatch = new Stopwatch();
-   private double floatTime = 0.0;
-   private final static Stopwatch booleanWatch = new Stopwatch();
-   private double booleanTime = 0.0;
-
    private final Set<YOLOv8DetectionOutput> detections;
    private final MatVector outputBlobs;
    private final RawImage detectionImage;
    private final FloatIndexer outputMasksIndexer;
-   private final Map<String, Mat> objectMasks = new HashMap<>();
+   private final Map<YOLOv8DetectionOutput, Mat> objectMasks = new HashMap<>();
```

Comment:
When a mask gets calculated, it's stored internally so we don't have to re-calculate if it's requested again.